e2e/Forc.toml

```diff
@@ -110,3 +111,5 @@ members = [
   'sway/types/scripts/script_u256',
   'sway/types/scripts/script_vectors',
 ]
+[patch.'https://github.com/fuellabs/sway']
+std = { git = "https://github.com/fuellabs/sway", branch = "esdrubal/abi_changes" }
```

Comment:
We gotta keep an eye on this. Like it happened before, _when_ the sway branches gets merged and deleted, the user will see some weird bugs.

Reply:
Hopefully sway gets a release soon after we do so we can remove this.